llama_hub/file/markdown/base.py

```diff
@@ -106,5 +105,13 @@ def load_data(
         """Parse file into string.
         If content is provided, use that instead of reading from file."""
         tups = self.parse_tups(file, content=content)
+        results = []
         # TODO: don't include headers right now
-        return [Document(text=value, extra_info=extra_info or {}) for _, value in tups]
+        results = [
+            Document(
+                text=f"\n\n{header}\n{value}" if header else value, 
```

Comment:
nit: this would create an extra newline if there's no header.

Reply:
> nit: this would create an extra newline if there's no header.

Hi @Disiok , thanks for the review. A query though. If no header the code fetches only the value right. I tested with some samples and seem to work, Could you clarify, please?